lib/cdo/shared_constants.rb

```diff
@@ -743,4 +743,25 @@ module SharedConstants
     STUDENT: 'student',
     TEACHER: 'teacher',
   ).freeze
+
+  AI_REQUEST_EXECUTION_STATUS = {
```

Comment:
Are you naming as `AI_REQUEST...` instead of `AICHAT_REQUEST...` anticipating that these will also be used across AI projects?

Reply:
Yeah, that was my thought (and that `AICHAT_REQUEST_EXECUTION_STATUS` felt a bit too long 😄)